src/app/shared/document/document-detail/document-detail.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="row mt-2 mb-2">
+  <div class="col-8 d-flex align-bottom align-self-end text-nowrap ellipsis">
+    <div class="align-text-bottom">
+      <h4 i18n="@@document.detail.heading">Document detail</h4>
+      <span class="small"><i class="fas {{collection?.icon}}" [style.color]="collection?.color"></i>&nbsp;{{collection?.name}}</span>&nbsp;
+      <strong *ngIf="summary">{{summary}}</strong>
+      <em *ngIf="!summary" i18n="@@shared.document.detail.document.indistinguishable">Indistinguishable document</em>
+    </div>
+  </div>
+  <div class="col small text-right d-flex align-bottom notice">
+    <div class="justify-content-end w-100">
+      Version {{_documentModel.dataVersion}}<br/>
+      Created on {{getNativeDate(_documentModel.creationDate) | date : 'mediumDate'}} by {{createdBy$ | async}}<br/>
+      Last modified on {{getNativeDate(_documentModel.updateDate) | date : 'mediumDate'}} by {{updatedBy$ | async}}
+    </div>
+  </div>
+</div>
+<ng-container *ngFor="let row of rows; let idx = index;">
```

Comment:
This is not the case. I only change the list when there is a change in the displayed document. Otherwise, all objects and references are the same and only their inner attributes change. So there are no unnecessary refreshes on the page. It works as it should right now.